src/psyclone/tests/domain/lfric/transformations/lfric_extract_test.py

```diff
@@ -748,7 +723,7 @@ def test_extract_kernel_and_builtin_lfric():
     # assert LFRicBuild(tmpdir).code_compiles(psy)
 
 
-def test_extract_colouring_omp_lfric():
+def test_extract_colouring_omp_lfric(tmpdir):
```

Comment:
I can't see how the `tmpdir` fixture is being used?

Reply:
Oh yes, I tried to enable compilation, but that failed (because the extract library is not compiled, otherwise it all works). Off topic for this PR, so I've removed tmdir again.